example/gluon/lipnet/models/network.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Maybe put this after the Apache license...

Reply:
Fixed